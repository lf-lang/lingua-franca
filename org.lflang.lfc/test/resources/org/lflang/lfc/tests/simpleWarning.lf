target unknown;

main reactor Foo {

}
