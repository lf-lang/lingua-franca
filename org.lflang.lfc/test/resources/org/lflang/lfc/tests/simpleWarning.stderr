lfc: error: Unrecognized target: unknown
--> %%%PATH.lf%%%:1:8
  |
1 | target unknown;
  |        ^^^^^^^ Unrecognized target: unknown
  |
2 |

lfc: error: Name of main reactor must match the file name (or be omitted).
--> %%%PATH.lf%%%:3:14
  |
2 |
3 | main reactor Foo {
  |              ^^^ Name of main reactor must match the file name (or be omitted).
  |
4 |
